app/2.0/nav.yaml

```diff
@@ -197,6 +194,11 @@
     path: /2.0/toolbox/server
```

Comment:
Move this doc to "Deploy" section (after "Optimize") and change the URL to "serve" (which matches your file name, below).

Add a link for PRPL pattern doc.

I assume in a subsequent PR you're going to replace the "Optimize for production" doc with your new "Build for production" doc.